test/unit/cog_api/http/relays_test.exs

```diff
@@ -190,5 +190,15 @@ defmodule CogApi.HTTP.RelaysTest do
         end
       end
     end
+
+    context "when the relay by name cannot be deleted" do
+      it "returns an error" do
+        cassette "relay_delete_failure_by_name" do
+          {:error, [error]} = Client.relay_delete(%{name: "nada"}, valid_endpoint)
+
+          assert error == "Resource not found: The relay nada could not be deleted"
```

Comment:
Should we quote or backtick the name? Something like: `Resource not found: The relay "nada" could not be deleted`